orbit/cmd/desktop/desktop.go

```diff
@@ -466,10 +453,40 @@ func setupLogs() {
 		MaxAge:     28, // days
 	}
 
-	log.Logger = log.Output(zerolog.MultiLevelWriter(
-		zerolog.ConsoleWriter{Out: logFile, TimeFormat: time.RFC3339Nano, NoColor: true},
-		stderrOut,
-	))
+	consoleWriter := zerolog.ConsoleWriter{Out: logFile, TimeFormat: time.RFC3339Nano, NoColor: true}
+	log.Logger = log.Output(consoleWriter)
```

Comment:
Why are we using both `zerolog` and `log` -- why are we using 2 logging packages here?

Reply:
`log` is a sub package of `zerolog`:
```
	"github.com/rs/zerolog"
	"github.com/rs/zerolog/log"
```